tests/python/lang/path_using_user_input/testdata/main.py

```diff
@@ -0,0 +1,18 @@
+user_upload_path = form.cleaned_data["filepath"]
+# bearer:expected python_lang_path_using_user_input
+os.mkdir(user_upload_path)
+
+print("What file would you like to read?")
+filepath = input() 
+# bearer:expected python_lang_path_using_user_input
+open(filepath)
+
+# bearer:expected python_lang_path_using_user_input
+with fileinput.input(files=(filepath), encoding="utf-8") as f:
```

Comment:
We currently don't catch this case, maybe because of "assignment" in paramteres. Will need to investigate further /cc @didroe maybe we could look together 🙏